### ISSUE TITLE
The app display blank after upgrade Blazor version 0.3.0

### ISSUE THREAD

**Issue by Smilefounder** [aspnet/Blazor#804]
Hi team,

My SPA app stops working and displays a blank page after upgrade to Blazor version 0.3.0. Could you help advise is there any configuration or additional migration steps required for upgrade to v0.3.0?

Repo: https://github.com/Swastika-IO/Swastika-IO-Admin-Blazor

Thanks.
Huy

**Comment by SIkebe** [comment]
You have to do some steps to upgrade.
https://blogs.msdn.microsoft.com/webdev/2018/05/02/blazor-0-3-0-experimental-release-now-available/ 
See "Upgrade an existing project to Blazor 0.3.0" section.

**Comment by SIkebe** [comment]
>* Install all of the required bits listed above.
>* Update your Blazor package and .NET CLI tool references to 0.3.0.
>* Remove any package reference to Microsoft.AspNetCore.Razor.Design as it is now a transitive dependency.
>* Update the C# language version to be 7.3. You may need to restart Visual Studio for this change to take effect.
>* Update component parameters to not be public and to add the [Parameter] attribute.
>* Update layouts to inherit from BlazorLayoutComponent and remove the implementation of ILayoutComponent including the Body property.

**Comment by Smilefounder** [comment]
Thanks @SIkebe let me try again then keep you posted.
